sphinx/environment.py

```diff
@@ -961,6 +962,29 @@ def process_downloads(self, docname, doctree):
 
     def process_images(self, docname, doctree):
         """Process and rewrite image URIs."""
+        def collect_candidates(imgpath, candidates):
+            globbed = {}
+            for filename in glob(imgpath):
+                new_imgpath = relative_path(path.join(self.srcdir, 'dummy'),
+                                            filename)
+                if filename.lower().endswith('.pdf'):
+                    if 'application/pdf' not in candidates:
+                        globbed.setdefault('application/pdf', []).append(new_imgpath)
```

Comment:
No, it continues collecting.
This pull request tries to search candidates twice (first is for `figure_language`, second is for original one). Line 971 is used for the second collecting to stop overriding if translated image already found.